src/lang/english.txt

```diff
@@ -1767,19 +1767,19 @@ STR_CONFIG_ERROR_SPRITECACHE_TOO_BIG                            :{WHITE}Allocati
 # Intro window
 STR_INTRO_CAPTION                                               :{WHITE}OpenTTD {REV}
 
-STR_INTRO_NEW_GAME                                              :{BLACK}New Game
-STR_INTRO_LOAD_GAME                                             :{BLACK}Load Game
-STR_INTRO_PLAY_SCENARIO                                         :{BLACK}Play Scenario
-STR_INTRO_PLAY_HEIGHTMAP                                        :{BLACK}Play Heightmap
-STR_INTRO_SCENARIO_EDITOR                                       :{BLACK}Scenario Editor
+STR_INTRO_NEW_GAME                                              :{BLACK}New game
+STR_INTRO_LOAD_GAME                                             :{BLACK}Load game
+STR_INTRO_PLAY_SCENARIO                                         :{BLACK}Play scenario
+STR_INTRO_PLAY_HEIGHTMAP                                        :{BLACK}Play heightmap
+STR_INTRO_SCENARIO_EDITOR                                       :{BLACK}Scenario editor
 STR_INTRO_MULTIPLAYER                                           :{BLACK}Multiplayer
 
-STR_INTRO_GAME_OPTIONS                                          :{BLACK}Game Options
-STR_INTRO_HIGHSCORE                                             :{BLACK}Highscore Table
+STR_INTRO_GAME_OPTIONS                                          :{BLACK}Options
```

Comment:
As discussed, until we actually know the "proper" difference between Game options & Settings, there needs to be something more that differentiates them - Put 'Game' back in for now

Reply:
This isn't changed since @LordAro's comment?

Reply:
Sorry guys, this one also slipped through while merging.